Add page-level PAGE interpretation

wrznr opened the pull request:
The new page-level `level` choice will not extract region or line
images but rather draw them onto the image file using
`PIL.draw.polygon`.
The PAGE XML namespace has also been updated.

wrznr committed:
Add page-level PAGE interpretation

The new page-level `level` choice will not extract region or line
images but rather draw them onto the image file using
`PIL.draw.polygon`.
The PAGE XML namespace has also been updated.

wrznr commented:
![image](https://user-images.githubusercontent.com/26707219/65143828-2831d180-da16-11e9-8bec-16d2d46dcd20.png)

tboenig reviewed (approved):
The solution for the new page schema looks good.

tboenig committed:
Update page2img.py

a litle typo ``GraphicsRegion``->``GraphicRegion``

tboenig committed:
Update page2img.py

I think it's better that the Zone has black borders. Then you can see the different zones better.
